### PR TITLE
Fix issue 117

### PR DESCRIPTION
**Pull request by NgoHuy**
I tested on my IMac and Macbook Air for supporting brew cask

### CODE REVIEW

File: lib/homebrew.sh [hunk 1]
```diff
@@ -96,7 +96,31 @@ homebrew_Rs() {
 }
 
 homebrew_R() {
-  brew remove "$@"
+  brew remove "$@" 2>&1 \
+  | awk '
+      BEGIN {
+        cask = 0;
```
Comment (icy) [inline lib/homebrew.sh]: @NgoHuy this variable is not used, did you mean `may_cask`?

File: lib/homebrew.sh [hunk 2]
```diff
@@ -96,7 +96,31 @@ homebrew_Rs() {
...
+      }
+      {
+        print;
+        if ($0 ~ /Error: No such keg:/) {
+          may_cask = 1;
```
Comment (icy) [inline lib/homebrew.sh:107]: maybe assign `may_cask = 217` and then `exit may_cask` in the last step. Save a check ;)

File: lib/homebrew.sh [hunk 3]
```diff
@@ -150,5 +174,29 @@ homebrew_Sccc() {
...
-  brew install $_TOPT "$@"
+  brew install $_TOPT "$@" 2>&1 \
+  | awk '
+      BEGIN {
+        cask = 0;
+      }
+      {
+        print;
+        if ($0 ~ /Found a cask named/) {
+          cask = 1;
```
Comment (icy) [inline lib/homebrew.sh]: I'd suggest to use `cask = 217`. Btw, can we use the same variable name as in the previous method? Either `may_cask` or `cask`, but not both please ;)

File: lib/homebrew.sh [hunk 4]
```diff
@@ -150,5 +174,29 @@ homebrew_Sccc() {
...
+      END {
+        if (cask == 1) {
+          exit 217;
+        }
+      }
+    '
+  _ret=( ${PIPESTATUS[*]} )
+
+  if [[ "${_ret[1]}" == "217" ]]; then
+    echo >&2 ":: (pacapt) fall back on cask"
```
Comment (icy) [inline lib/homebrew.sh:199]: `fall back to`, Imho

File: lib/homebrew.sh [hunk 5]
```diff
@@ -96,7 +96,31 @@ homebrew_Rs() {
...
+      END {
+        if (may_cask == 1) {
+          exit 217;
+        }
+      }
+    '
+  _ret=( ${PIPESTATUS[*]} )
+
+  if [[ "${_ret[1]}" == "217" ]]; then
+    echo >&2 ":: (pacapt) fallback on cask"
```
Comment (icy) [inline lib/homebrew.sh:119]: `fall back to`. `fall back on` means `rely on`.

File: lib/homebrew.sh (hunk 4 above)
Comment (cuonglm) [reply lib/homebrew.sh:199]: And why not use `-eq`?

**Comment by NgoHuy** [comment]
the original fix was written by Duy. Then I close this pull for fair play with him.

**Pull request closed without merging**